### DISCUSSION TITLE
Applied and got approved but no PRO

### DISCUSSION THREAD

**Discussion by albersonmiranda** [community/community#30773] [GitHub Education]
I've applied for Github Student Developer Pack and got approved but PRO isn't activated (no protected branch etc). Is there something I should do?

**Comment by rfals** [comment]
Same here, did u manage to fix it?

**Reply by albersonmiranda** [reply]
Still no pro features. Opened a ticket. Two actually. They closed both without solution.

**Comment by rfals** [comment]
I also opened a ticket and wrote that in addition to not having Pro features I also don't have access to Copilot. They just responded that I have access to Copilot, even though I don't, and closed the ticket lol.
